implementation/src/main/java/io/smallrye/jwt/auth/cdi/PrincipalProducer.java

```diff
@@ -60,6 +57,7 @@ public String getName() {
         }
 
         @Override
+        @SuppressWarnings("squid:S1168") // Indicate to Sonar that a null return is acceptable
```

Comment:
I'd avoid getting these Sonar specific indicators into the code, if some other system  we only need to address the real code issues, not a problem if Sonar reports some false alarms

Reply:
Noted - I'll remove the four occurrences of them.

Reply:
Pushed the removal of Sonar warning suppression.